tests/queries/0_stateless/03275_subcolumns_in_primary_key.reference

```diff
@@ -86,7 +86,7 @@ Sorting: a.b.c DESC, a.b.d DESC
 0	((0,'str_0',[]))	Hello, World!
 2	((0,'str_2',[0,1]))	Hello, World!
 Expression ((Project names + Projection))
-Filter ((WHERE + Change column names to column identifiers))
```

Comment:
Why are filters now replaced with expressions?

Reply:
Because when condition is moved to prewhere we don't have this filter step: https://fiddle.clickhouse.com/07e08ac2-b1d3-4a13-845a-d38980cfa6a2
In this test all conditions were with subcolumns and prewhere didn't work. Now it works and the result of explain changed